docs/creating-bots/javascript/unit-testing.mdx

```diff
@@ -1,22 +1,22 @@
 ---
-sidebar_label: 'Validation Bots'
+sidebar_label: 'Unit Testing'
 ---
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-# Defining Bots for Validation Testing
+# Defining Bots for Unit Testing
 
 Your Unity integration collects state information about GameObjects and notifies the Bot runtime with the updated state at a fixed tick interval.
 The game state can be monitored to test the impacts of GameObject behaviors on other GameObjects.
 
-Validation Bots share the same APIs as [PlayTest Bots](./playtest-bots), with two important distinctions:
+Validation Bots share the same APIs as [Dynamic Bots](./dynamic-decision-making), with two important distinctions:
```

Comment:
Maybe we should also talk about "Dynamic" bots - all of our bots are dynamic (i.e. they receive state and perform actions), rather than something that plays back input which is what I would call not dynamic.